buzzmobile/constants.yaml

```diff
@@ -44,7 +44,7 @@ wheel_separation: 1.3716 # distance between left and right wheels in meters
 max_steering_angle: 0.262 # radians (=15 degrees)
 
 # Max speed of buzzmobile in m/s
-max_speed: 0.2
+max_speed: 2 #TODO this is a murder machine
```

Comment:
I'm bumping it down to 1 m/s now as a compromise. Still a bit faster than the car but if we set it to 0.2 then the simulation (running at 0.03 real time factor) doesn't let you see the car move.